packages/react/src/components/StructuredList/StructuredList.js

```diff
@@ -173,41 +178,39 @@ StructuredListRow.propTypes = {
    */
   head: PropTypes.bool,
 
-  /**
-   * Specify whether a `<label>` should be used
-   */
-  label: PropTypes.bool,
```

Comment:
Removing props is totally fine I think, just need to use the `deprecate` prop helper first so that people who use this prop now won't have errors about these props being passed on to DOM nodes through `other`